src/libraries/Common/tests/System/Net/Security/Kerberos/FakeKerberosPrincipal.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using Kerberos.NET.Crypto;
+using Kerberos.NET.Entities;
+using Kerberos.NET.Server;
+
+namespace System.Net.Security.Kerberos;
+
+class FakeKerberosPrincipal : IKerberosPrincipal
+{
+    private readonly byte[] _password;
```

Comment:
nit: I would change the name to decrease chance we get flagged by security scans.

Reply:
RFC-wise it's called "long term secret"